clang/lib/CConv/RewriteUtils.cpp

```diff
@@ -224,22 +224,28 @@ class TypeExprRewriter
   Rewriter &Writer;
 
   void rewriteType(Expr *E, SourceRange &Range) {
-    CVarSet CVSingleton = Info.getPersistentConstraintVars(E, Context);
+    if (!Info.hasPersistentConstraints(E, Context))
+      return;
```

Comment:
Another place that something doesn't seem like it should happen, but does?

Reply:
Previously non-pointer types got empty constraint sets, now they don't get any entry in the map.